onboarding/computer-setup.md

```diff
@@ -154,3 +154,35 @@ export PROJECT_HOME=$HOME/
 source /usr/local/bin/virtualenvwrapper.sh #centos
 source /usr/share/virtualenvwrapper/virtualenvwrapper.sh #ubuntu
 ```
+
+## Email
```

Comment:
i think you can put back the emoji, it looks good

Reply:
Done in https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/78/commits/97e73316c5eaef75c096e9da562b305a5ae55a18